manager/manager.go

```diff
@@ -491,6 +491,10 @@ func (m *Manager) Run(parent context.Context) error {
 	healthServer.SetServingStatus("Raft", api.HealthCheckResponse_NOT_SERVING)
 	localHealthServer.SetServingStatus("ControlAPI", api.HealthCheckResponse_NOT_SERVING)
 
+	if err := m.watchServer.Start(ctx); err != nil {
```

Comment:
Non-blocking:  probably doesn't matter much, but would it make sense to start this after the raft node has started up and joined (when the other watches are set)?  Otherwise if something starts watching right away, would they get the events from the raft node starting up and loading all of its state from disk?